lib/components/molecules/Typeahead/types/index.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import type { Node } from 'react';
+
+export type Props = {
+  children: Node,
+  id: string,
+  name: string,
+  type: string,
+  disabled: boolean,
+  children: Node,
+  getSuggestions?: any,
+  suggestions: any,
+  inheritedStyles?: string,
+  className: string,
+  renderSuggestion: any,
+  placeholder: string,
+  value: any,
```

Comment:
The value any is wrong, its almost like not defining a type.

Reply:
done